### DISCUSSION TITLE
How to get  alternative port in module

### DISCUSSION THREAD

**Discussion by max5432112345** [nuxt/nuxt#17019] [General]
> [get-port] Unable to find an available port (tried 3000). Using alternative port: 3001                                                                                               
 
That return port 3000
const defaultPort = process.env.API_PORT || moduleOptions.port || process.env.PORT || process.env.npm_package_config_nuxt_port || nuxt.options.server && nuxt.options.server.port;

**Comment by max5432112345** [comment]
In nuxt.options.server.port (deprecated) is 3000 not 3001

**Reply by pi0** [reply]
Please try latest edge release. It should be synced now.

**Reply by max5432112345** [reply]
https://github.com/nuxt/framework/discussions/6558#discussioncomment-3463407

**Comment by max5432112345** [comment]
> process.env.API_PORT || moduleOptions.port || process.env.PORT || process.env.npm_package_config_nuxt_port ||

@pi0 console.log(nuxt.options.server.port) return 3000 not 3001 in defineNuxtModule
